packages/ketcher-react/src/hooks/useRaf.ts

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import { useEffect, useRef } from 'react';
+
+export const useRaf = (active: boolean, onFrame: (dtMs: number) => void) => {
```

Comment:
Let's not use shortens here. Just useRequestAnimationFrame, requestAnimationFrameId and deltaTimeInMilliseconds would be better because nobody will be lost after seeing it first time without additional context